src/siteconfig/forms.py

```diff
@@ -143,3 +144,9 @@ def clean_deck_owner(self):
             deck_owner.is_superuser = True
             deck_owner.save()
         return deck_owner
+
+    def clean_custom_profile_field(self):
+        """
+        Ensure custom_profile_field with only whitespaces is not valid
```

Comment:
Confusing docstring.  I think you mean Ensure it IS valid?  better, say you are removing the whitespace to ensure it is still valid?  Or... I'm not sure exactly.  Just spell out why you are doing this.

Reply:
Its mostly there to make sure an empty string like '      ' isn't valid. Therefore wont show up in some form fields.

updated the doc to this:
```python
"""
Format's custom_profile field and ensures its not completely composed of whitespace.
ie. 
- '  Grad Year' => 'Grad Year'
- '     ' => ''  (which means custom_profile_field wont show as its empty)
"""
```

also added a test for the method